contributing/component-validator-rules.md

```diff
@@ -457,9 +457,13 @@ Optionally, you may add more than one valid variant and/or invalid examples.
 
 ## Test Output files
 
-For each test file, also add a matching output file which will display the
-validator output for the test case. If you only added valid examples, this file
-should contain a single line:
+After creating your validator html file, You need to create the corresponding `.out` file to act as your test results which is used to verify if the validator is still validating correctly for your extension. Todo this, navigate to the root of the project, `amphtml/` and run `gulp validator --update_tests`. This shoudl generate a matching `.out` file, and for this example, it would be:
```

Comment:
Could you split the sentences?

From:

`After creating your validator html file, You need to create the corresponding `.out` file to act as your test results which is used to verify if the validator is still validating correctly for your extension.`

To:

`After creating your validator html file, You need to create the corresponding `.out` file to act as your test results. These are used to verify the validator is validating your extension correctly.`

Also I think `todo` in this context should be `to do` and `shoudl` should be `should` ;).